scrapy/downloadermiddlewares/cookies.py

```diff
@@ -71,7 +72,10 @@ def _debug_set_cookie(self, response, spider):
 
     def _format_cookie(self, cookie):
         # build cookie string
-        cookie_str = '%s=%s' % (cookie['name'], cookie['value'])
```

Comment:
It also works with `cookie_str = to_bytes('%s=%s' % (cookie['name'], to_native_str(cookie['value'])))` (see https://github.com/elacuesta/scrapy/commit/9a74d4257dea86fb72271edfd68ba750e9a2b995)